ACTIVITI-5522 preview release for 8.1.4

arpanhyland opened the pull request:
<!--
     For Work In Progress Pull Requests, please use the Draft PR feature,
     see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.

     Before submitting a Pull Request, please ensure you've done the following:
     - 📖 The commit message follows our guidelines
     - 👷‍♂️ Tests for the changes have been added (for bug fixes / features)
     - 👷‍♀️ Create small PRs. In most cases this will be possible.
     - ✅ Provide tests for your changes.
     - 📝 Use descriptive commit messages.
     - 📗 Update any related documentation and include any relevant screenshots.
-->

<!--
(check one with "x") one of the following
-->

**What kind of change does this PR introduce?**

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [X ] Other... Please describe:

## Description
Triggering preview release for 8.1.4
<!--
You can also link to an open issue here
-->

- Issue Link:

<!--
If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...
-->

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

arpanhyland committed:
preview release with database fix

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/Activiti/Activiti?pullRequest=5116) <br/>All committers have signed the CLA.

arpanhyland committed:
Merge branch '8.1.x' into ab-ACTIVITI-5522-preview-release

arpanhyland committed:
Removed duplicate index

ShreasiK committed:
Remove creation and deletion of index ACT_IDX_IDENT_LNK_TASK for oracle as another index ACT_IDX_TSKASS_TASK already present for the same column
Added if exists before dropping the index ACT_IDX_IDENT_LNK_TASK & ACT_IDX_RE_PROCDEF_DEPLOYMENT_ID

ShreasiK committed:
Added code -> drop index if exists else do nothing for mysql & oracle

ShreasiK committed:
Solution for EOF error

arpanhyland committed:
modified exception handling if no index found

arpanhyland committed:
changing statements to lowercase

balsarori reviewed (approved):
LGTM
